website/docs/intro.md

```diff
@@ -0,0 +1,41 @@
+---
+sidebar_position: 1
+title: Introduction
+---
+
+### What is odo?
+
+odo is a fast, iterative and straightforward CLI tool for developers who write, build, and deploy applications on Kubernetes.
+
+odo abstracts the complex Kubernetes terminology so that an application developer can focus on writing code in their favourite framework without having to learn Kubernetes.
+
+odo is focussed on inner loop development with some tooling that would help users transition to the outer loop.
```

Comment:
focused

Reply:
An intro to inner/outer loop would be nice

Reply:
Added it from [this source](https://developers.redhat.com/blog/2020/06/16/enterprise-kubernetes-development-with-odo-the-cli-tool-for-developers#improving_the_developer_workflow). Surprisingly, it was the first Google search result for "software inner loop vs. outer loop"!